src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,82 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.exception.DukeIOException;
+
+/**
+ * The class that is responsible for reading and writing raw taskList data.
+ */
+public class Storage {
+
+    private final File f;
```

Comment:
Can perhaps use a more descriptive variable name, such as file, since it has a rather large scope?